test/powershell/Language/Parser/RedirectionOperator.Tests.ps1

```diff
@@ -29,13 +29,14 @@ Describe "Redirection operator now supports encoding changes" -Tags "CI" {
         $psdefaultParameterValues.Remove("out-file:encoding")
     }
 
-    It "If encoding is unset, redirection should be Unicode" {
+    It "If encoding is unset, redirection should be UTF8 without bom" {
         $asciiString > TESTDRIVE:\file.txt
         $bytes = get-content -encoding byte TESTDRIVE:\file.txt
-        # create the expected
-        $BOM = [text.encoding]::unicode.GetPreamble()
-        $TXT = [text.encoding]::unicode.GetBytes($asciiString)
-        $CR  = [text.encoding]::unicode.GetBytes($asciiCR)
+        # create the expected - utf8 encoding without a bom
+        $encoding = [text.utf8encoding]::new($false)
+        $BOM = $encoding.GetPreamble()
```

Comment:
This is very misleading; the test is verifying with a BOM yet the test is requesting the BOM.  If the Encoding class has a bug or was created incorrectly, the test will still pass.  In other words, if the array is empty, as expected, the test passes. If the array is non-empty, the test will still pass.

I suggest removing $BOM completely and not relying on the implicit empty array.

Reply:
done